extra_geom/detectors.py

```diff
@@ -1342,3 +1346,134 @@ def __init__(self, modules, filename='No file'):
         warnings.warn(
             "DSSC_Geometry has been renamed to DSSC_1MGeometry.", stacklevel=2
         )
+
+
+class JUNGFRAUGeometry(DetectorGeometryBase):
+    """Detector layout for flexible Jungfrau arrangements based on the JF-500K
+       module, which is an independent detector unit of 2 x 4 ASIC tiles.
+
+       The ss/fs assignment is the other way around as for AGIPD:
```

Comment:
Nitpick: let's remove the reference to AGIPD - there's no particular reason someone looking at JUNGFRAU geometry should be familiar with AGIPD.

Also, let's spell out slow-scan & fast-scan in the docstring (even if I haven't done this everywhere ;-) ). The abbreviations aren't familiar to everyone, and I know people who prefer to avoid the initials 'ss' even in innocent contexts.

Reply:
This has been corrected with the last commit